descriptors/mongodb/debezium-mongodb-2.0.1.Final.json

```diff
@@ -1,6 +1,6 @@
 {
   "connector_type" : {
-    "id" : "debezium-mongodb-2.0.1.Final",
+    "id" : "debezium-mongodb",
```

Comment:
FYI @dhirajsb  @valdar  This change should finally fix the cos-fleetmanager issue with old/legacy Debezium connector-types still showing up after a new release.